operator/v1/types_baremetal.go

```diff
@@ -0,0 +1,68 @@
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
```

Comment:
The enhancment talks about creating these only in one namespace.  Why have a namespaced object that can only be created in a single namespace by a privileged user?

Reply:
Agreed. This can be made Non-namespaced.

Reply:
Yes. At the time of writing the enhancement proposal, it appeared that having the CR Namespaced was not required but harmless.

Reply:
update to make non-namespaced